x/ecocredit/server/testsuite/genesis.go

```diff
@@ -186,11 +205,158 @@ func (s *IntegrationTestSuite) assetClassInfoEqual(q, other *ecocredit.ClassInfo
 	require.Equal(q.Metadata, other.Metadata)
 }
 
-func (s *IntegrationTestSuite) assetBatchInfoEqual(q, other *ecocredit.BatchInfo) {
+func (s *IntegrationTestSuite) assetProjectInfoEqual(q, other *ecocredit.ProjectInfo) {
 	require := s.Require()
 	require.Equal(q.ClassId, other.ClassId)
-	require.Equal(q.BatchDenom, other.BatchDenom)
+	require.Equal(q.ProjectId, other.ProjectId)
+	require.Equal(q.ProjectLocation, other.ProjectLocation)
+	require.Equal(q.Metadata, other.Metadata)
 	require.Equal(q.Issuer, other.Issuer)
+}
+
+func (s *IntegrationTestSuite) assetBatchInfoEqual(q, other *ecocredit.BatchInfo) {
```

Comment:
you can use Go's `==` operator on structs of the same type, so if we wanted to slim this down we could do

`require.True(q == other)` or `require.Equal(q, other)`